CMakeLists.txt

```diff
@@ -146,37 +148,36 @@ add_subdirectory(test)
 set(GLM_INSTALL_CONFIGDIR "${CMAKE_INSTALL_LIBDIR}/cmake/glm")
 install(DIRECTORY glm DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 
-if (NOT CMAKE_VERSION VERSION_LESS "2.8.12")
-	include(CMakePackageConfigHelpers)
-	write_basic_package_version_file(
-	    "${CMAKE_CURRENT_BINARY_DIR}/glmVersion.cmake"
-	    VERSION ${GLM_VERSION}
-	    COMPATIBILITY AnyNewerVersion
-	)
-
-	# build tree package config
-	configure_file(
-	    cmake/glmBuildConfig.cmake.in
-	    glmConfig.cmake
-	    COPYONLY
-	)
-
-	# install tree package config
-	configure_package_config_file(
-	    cmake/glmConfig.cmake.in
-	    ${GLM_INSTALL_CONFIGDIR}/glmConfig.cmake
-	    INSTALL_DESTINATION ${GLM_INSTALL_CONFIGDIR}
-	    PATH_VARS CMAKE_INSTALL_INCLUDEDIR
-	    NO_CHECK_REQUIRED_COMPONENTS_MACRO
-	)
-
-	install(
-	    FILES 
-	        "${CMAKE_CURRENT_BINARY_DIR}/glmConfig.cmake"
-	        "${CMAKE_CURRENT_BINARY_DIR}/glmVersion.cmake"
-	    DESTINATION ${GLM_INSTALL_CONFIGDIR}
-	)
+write_basic_package_version_file(
+    "${CMAKE_CURRENT_BINARY_DIR}/glmVersion.cmake"
+    VERSION ${GLM_VERSION}
+    COMPATIBILITY AnyNewerVersion
+)
+
+# build tree package config
+configure_file(
+    cmake/glmBuildConfig.cmake.in
+    glmConfig.cmake
+    @ONLY
+)
+
+# install tree package config
+configure_package_config_file(
+    cmake/glmConfig.cmake.in
+    ${GLM_INSTALL_CONFIGDIR}/glmConfig.cmake
+    INSTALL_DESTINATION ${GLM_INSTALL_CONFIGDIR}
+    PATH_VARS CMAKE_INSTALL_INCLUDEDIR
+    NO_CHECK_REQUIRED_COMPONENTS_MACRO
+)
+
+install(
+    FILES 
+        "${CMAKE_CURRENT_BINARY_DIR}/${GLM_INSTALL_CONFIGDIR}/glmConfig.cmake"
+        "${CMAKE_CURRENT_BINARY_DIR}/glmVersion.cmake"
+    DESTINATION ${GLM_INSTALL_CONFIGDIR}
+)
 
+if (NOT CMAKE_VERSION VERSION_LESS "2.8.12")
```

Comment:
This does not work for me on Ubuntu with cmake version 2.8.12.2:

    CMake Error at CMakeLists.txt:186 (install):
      install TARGETS given no ARCHIVE DESTINATION for static library target
      "glm".

Probably because this version number is actually greater than "2.8.12"?